skimage/restoration/deconvolution.py

```diff
@@ -364,10 +367,14 @@ def richardson_lucy(image, psf, iterations=50, clip=True):
     image = image.astype(np.float)
     psf = psf.astype(np.float)
     im_deconv = np.full(image.shape, 0.5)
-    psf_mirror = psf[::-1, ::-1]
+    psf_mirror = np.flip(psf)
```

Comment:
was this a bug which you corrected?

Reply:
Yes, this was the "Change PSF flip to work in ndim!=2" part of the description